.github/workflows/e2e.storages.yaml

```diff
@@ -38,12 +38,16 @@ jobs:
       matrix:
         config-file:
           - storage/h2/e2e.yaml
-          - storage/opensearch/e2e.yaml
           - storage/mysql/e2e.yaml
           - storage/tidb/e2e.yaml
           - storage/influxdb/e2e.yaml
           - storage/postgres/e2e.yaml
         include:
+          - os-version: 1.1.0
```

Comment:
os as shortcut of OpenSearch seems a little tricky, os usually is for operation system.

Reply:
let's keep opensearch as full word?